js-challenge/weather-api/index.js

```diff
@@ -0,0 +1,56 @@
+
+function areaWhether() {
+    //プルダウンの情報を受け取る
+    var city = document.getElementById("city").value;
+    var text = '';
+    if (city === 'London') {
+        currentWhether();
+    }
+    else if (city === 'Tokyo') {
+        currentWhether();
+    }
+    else if (city === 'Paris') {
+        currentWhether();
+    }
+    else if (city === 'New York') {
+        currentWhether();
+    }
+    else if (city === 'Indonesia') {
+        currentWhether();
+    }
+};
+
+//Ajaxで情報を取得
+function currentWhether() {
+
+    var city = document.getElementById("city").value;
+    var api_key = "4b5774e9f3d2a07b84f0f2f88e486224";
+    var url = "http://api.openweathermap.org/data/2.5/weather?q=" + city + "&appid=" + api_key + "&lang=ja&units=metric";
+    var xhr = new XMLHttpRequest();
+    xhr.open("GET", url);
+    xhr.send();
+
+    xhr.onreadystatechange = function () {
```

Comment:
functionも古いのでアロー関数を使いましょう。

varやfunctionには不都合な挙動があるので、新しい文法(var,let,アロー関数,etc...)に置き換わったらしいです。

```suggestion
    xhr.onreadystatechange = () => {
```